tests/metagpt/document_store/test_qdrant_store.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2023/6/11 21:08
+@Author  : alexanderwu
+@File    : test_milvus_store.py
+"""
+import random
+
+from qdrant_client.models import PointStruct, Distance, VectorParams, FieldCondition, Range, Filter
+
+from metagpt.document_store.qdrant_store import QdrantConnection, QdrantStore
+from metagpt.logs import logger
+
+vectors = [[random.random() for _ in range(2)] for _ in range(10)]
+
+points = [
+    PointStruct(
+        id=idx,
+        vector=vector,
+        payload={"color": "red", "rand_number": idx % 10}
+    )
+    for idx, vector in enumerate(vectors)
+]
+
+
+def test_milvus_store():
+    qdrant_connection = QdrantConnection(memory=True)
+    vectors_config = VectorParams(size=2, distance=Distance.COSINE)
+    qdrant_store = QdrantStore(qdrant_connection)
+    qdrant_store.create_collection("Book", vectors_config, force_recreate=True)
```

Comment:
If using `force_recreate=True`, to check if the `Book` collection' data updated?

Reply:
Ok